### ISSUE TITLE
CustomDialog in Visual Studio Designer throws exception

### ISSUE THREAD

**Issue by marcpiechura** [MahApps/MahApps.Metro#1850]
HI, 

I would like to see the preview of my CustomDialog in Visual Studio but unfortunately it shows the folling error message.

![image](https://cloud.githubusercontent.com/assets/11612539/6802297/82c9439c-d22f-11e4-8755-855c3cb2c4e4.png)

The fix is realy simple, in BaseMetroDialog -> HandleTheme add the NULL check for windowTheme

```

                var windowTheme = DetectTheme(this);

	            if (windowTheme == null)
		            return;


                var theme = windowTheme.Item1; <-- Throws
```

Thanks!

**Comment by johnsimmons** [comment]
I apologize if this is the incorrect place to post a question, but I didn't want another duplicate of this problem (I'll remove it and ask somewhere else if needed).  

I'm having this same problem with the newest version of MahApps.Metro.  The mentioned code fix is in place but my problem is identical to the original post by marcpiechura.

I added a ```UserControl (WPF)``` and added this code to my new xaml file

```<Dialogs:CustomDialog x:Class="HR_DesktopApplication.MyCustomDialog"
             xmlns:Dialogs="clr-namespace:MahApps.Metro.Controls.Dialogs;assembly=MahApps.Metro"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
             mc:Ignorable="d" 
             d:DesignHeight="300" d:DesignWidth="300">
    <Grid>
        
    </Grid>
</Dialogs:CustomDialog>```

I imagine this is human error (since I'm new to wpf in general), but I wanted to make sure it wasn't the same bug.

**Comment by marcpiechura** [comment]
@johnsimmons 
You need to upgrade to the latest 1.1.3 alpha, it's not included in 1.1.2. 

@punker76 
Maybe you could change the milestone to 1.1.3 for clarification ?

Regards

**Comment by johnsimmons** [comment]
Awesome, thanks for the response.  I'm glad it was just a misunderstanding on my part.
